spec/lib/gakubuchi/engine_registrar_spec.rb

```diff
@@ -35,22 +39,31 @@
         expect(described_method.call).to eq false
       end
 
-      it "shouldn't register the engine for the MIME type" do
+      it "shouldn't register the MIME type with Sprockets::Environment" do
+        expect(&described_method).not_to change { env.mime_types }
+      end
+
+      it "shouldn't register the engine with Sprockets::Environment" do
         expect(&described_method).not_to change(&sprokcets_extensions)
       end
     end
 
     context "when all parameters are valid" do
       let(:content_type) { "application/csv+ruby" }
       let(:engine) { Tilt::CSVTemplate }
-      let(:extensions) { %w(.rcsv .csv.ruby) }
+      let(:extensions) { %w(.rcsv .csv.rcsv) }
       let(:extensions_with_single_dot) { extensions.select { |ext| ext =~ /\A\.[^\.]+\z/ } }
 
       it "should return true" do
         expect(described_method.call).to eq true
       end
 
-      it "should register the engine for the MIME type" do
+      it "should register the MIME type with Sprockets::Environment", skip: major_version_of(Sprockets) < 4 do
```

Comment:
__[rubocop]__
* Line is too long. [110/100] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)

<!-- Comment by [SideCI](https://sideci.com) -->